### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by lwkoen**
- Status: workaround
  - Version: All versions 
  - Links: https://my.geocat.net/knowledgebase/125/Log4j-RCE-CVE-2021-44228-vulnerability-patch.html

### CODE REVIEW

File: software/README.md [hunk 1]
```diff
@@ -68,6 +68,19 @@ The `Version` relates to the `Status` column. If `Status` is Vulnerable, Version
...
+| Aruba Networks  | ArubaOS-CX switches | Not vuln |  | |[source](https://www.arubanetworks.com/assets/alert/ARUBA-PSA-2021-019.txt) |
+| Aruba Networks  | ArubaOS SD-WAN Controllers and Gateways | Not vuln |  | |[source](https://www.arubanetworks.com/assets/alert/ARUBA-PSA-2021-019.txt) |
+| Aruba Networks  | ArubaOS-S switches | Not vuln |  | |[source](https://www.arubanetworks.com/assets/alert/ARUBA-PSA-2021-019.txt) |
+| Aruba Networks  | ArubaOS Wi-Fi Controllers and Gateways | Not vuln |  | |[source](https://www.arubanetworks.com/assets/alert/ARUBA-PSA-2021-019.txt) |
+| Aruba Networks  | Aruba User Experience Insight (UXI) | Not vuln |  | |[source](https://www.arubanetworks.com/assets/alert/ARUBA-PSA-2021-019.txt) |
+| Aruba Networks  | Aruba VIA Client | Not vuln |  | |[source](https://www.arubanetworks.com/assets/alert/ARUBA-PSA-2021-019.txt) |
+| Aruba Networks  | Silver Peak Orchestrator | Workaround |  | |[source](https://www.arubanetworks.com/assets/alert/ARUBA-PSA-2021-019.txt), [workaround](https://www.arubanetworks.com/website/techdocs/sdwan/docs/advisories/media/security_advisory_notice_apache_log4j2_cve_2021_44228.pdf) |
```
Comment (maertsen) [inline software/README.md:83]: Unrelated change, but much appreciated :-)

**Review by maertsen — CHANGES_REQUESTED** [review]
hi @lwkoen, thanks for your contributions. Could you have a look at the review comments? Thanks!

File: software/README.md [hunk 2]
```diff
@@ -741,14 +754,15 @@ The `Version` relates to the `Status` column. If `Status` is Vulnerable, Version
...
 | GitHub | Github Enterprise Server | 3.3.1, 3.2.6, 3.1.14, 3.0.22 | Fix | | [source](https://github.blog/2021-12-13-githubs-response-to-log4j-vulnerability-cve-2021-44228/) |
 | GitLab | GitLab || Not vuln | | [source](https://forum.gitlab.com/t/cve-2021-4428/62763/8)|
 | GoAnywhere| Agents| Unknown| Workaround | | [source](https://www.goanywhere.com/cve-2021-44228-goanywhere-mitigation-steps)|
 | GoAnywhere| Gateway| Unknown| Workaround | | [source](https://www.goanywhere.com/cve-2021-44228-goanywhere-mitigation-steps)|
 | GoAnywhere| MFT| Unknown| Workaround | | [source](https://www.goanywhere.com/cve-2021-44228-goanywhere-mitigation-steps)|
-| Graylog | Graylog | < 3.3.15,<4.0.14,<4.1.9,<4.2.3 | Fix | The vulnerable Log4j library is used to record GrayLog's own log information. Vulnerability is not triggered when GrayLog stores exploitation vector from an outer system. | [source](https://www.graylog.org/post/graylog-update-for-log4j)|
+| Graylog | Graylog | < 3.3.15,<4.0.14,<4.1.9,<4.2.3 | Fix | | [source](https://www.graylog.org/post/graylog-update-for-log4j)|
```
[the remark this answers is not in the document]
Comment (lwkoen) [reply software/README.md:765]: I think we have a versioning problem. I didn't remove this line

File: software/README.md (hunk 1 above)
Comment (lwkoen) [reply software/README.md:83]: I didn't add this line

File: software/README.md [hunk 3]
```diff
@@ -863,7 +877,6 @@ The `Version` relates to the `Status` column. If `Status` is Vulnerable, Version
 | HPE |SGI UV 2000 Server| | Not vuln| |  [source](https://support.hpe.com/hpesc/public/docDisplay?docLocale=en_USdocId=a00120086en_us) |
 | HPE |SGI UV 3000 Server| | Not vuln| |  [source](https://support.hpe.com/hpesc/public/docDisplay?docLocale=en_USdocId=a00120086en_us) |
 | HPE |SGI UV 300, 300H, 300RL, 30EX| | Not vuln| |  [source](https://support.hpe.com/hpesc/public/docDisplay?docLocale=en_USdocId=a00120086en_us) |
-| HPE | Silver Peak Orchestrator | Workaround |  | |[source](https://www.arubanetworks.com/assets/alert/ARUBA-PSA-2021-019.txt), [workaround](https://www.arubanetworks.com/website/techdocs/sdwan/docs/advisories/media/security_advisory_notice_apache_log4j2_cve_2021_44228.pdf) |
```
[the remark this answers is not in the document]
Comment (lwkoen) [reply software/README.md]: re-added in my second commit

**Pull request closed without merging**